src/docs/reference/About.js

```diff
@@ -3,21 +3,131 @@
+  },
+  {
+    name: 'Chris Carlozzi',
+    avatar: '/docs/img/team/grommet-peep-chris.svg',
+    twitter: 'https://twitter.com/cclozzi',
+    github: 'https://github.com/L0ZZI'
+  },
+  {
+    name: 'Eric Soderberg',
+    avatar: '/docs/img/team/grommet-peep-eric.svg',
+    twitter: 'https://twitter.com/ericsoderberg',
+    github: 'https://github.com/ericsoderberghp'
+  },
+  {
+    name: 'Tracy Barmore',
+    avatar: '/docs/img/team/grommet-peep-tracy.svg',
+    twitter: 'https://twitter.com/tracybarmore',
+    github: 'https://github.com/tracybarmore'
+  }
+];
 
 export default class About extends Component {
 
   render () {
     return (
       <DocsArticle title='About'>
-
-        <section>
-          <Paragraph>
-            Grommet is about mixing fresh, pragmatic user interface design
-            with an implementation library and resources in an open
-            community.
+        <Section>
+          <Paragraph size="large">
+            Grommet came to the world from four individuals inside 
+            Hewlett Packard that wanted to make designing a modern 
+            web experience for enterprise companies (or anyone really) 
+            easy. Working day to day, the designer and developer 
+            handoff is always a point of contention when it 
+            comes to our productivity and ensuring a 
+            unique vision is delivered to 
+            customers—So that’s where we started.
           </Paragraph>
-        </section>
-
+          <Paragraph size="medium">
+            Our team used years of design and development experience 
+            to form Grommet. Taking the knowldge from various 
+            web technologies and customer focused experiences, 
+            we ventured out to the community to bootstrap with 
+            the best tools out there. Initially using 
+            Facebook’s React.js, Node.js, and Inuit as the 
+            underlying technologies, and layering on a 
+            visual design system and with some secret sauce to 
+            tie it all together. This provided the 
+            flexibility needed to create a dynamic library of components and 
+            allow Grommet to be the leading React UI 
+            Framework out there (horn toot).
+          </Paragraph>
+          <Paragraph size="medium">
+            We originally created Grommet to be a solution 
+            for our own organization’s needs. We quickly 
+            found the usefullness for a tool like this outside 
+            Hewlett Packard, and taking a note from our own 
+            founders, Dave and Bill, we shared it with the 
+            open source community with great response. As 
+            with any company, change is inevitabe, 
+            and change it did…Hewlett Packard became 
+            two companies. The split became proof that G
+            rommet was not only a valuable tool 
+            to the community, but to our own eccosystem.
```

Comment:
Oops, I had just copy and pasted this and didn't think to spell check.  I will do a spell check and resubmit this.